gridstatus/pjm.py

```diff
@@ -655,12 +666,20 @@ def get_interconnection_queue(self, verbose=False):
 
         return queue
 
-    def _get_key(self):
-        settings = self._get_json(
-            "https://dataminer2.pjm.com/config/settings.json",
-        )
+    def _get_api_key(self):
+        # Directly return the API key if it's already set
+        if self._api_key:
+            return self._api_key
+
+        # Retrieve the API key from environment variable if not set in the instance
+        env_api_key = os.environ.get("PJM_API_KEY")
+        if env_api_key:
+            return env_api_key
 
-        return settings["subscriptionKey"]
+        # If neither is set, raise an error
```

Comment:
What's the reasoning behind requiring the key versus falling back to the key in the settings file if not provided by the user?